Makefile

```diff
@@ -328,6 +328,9 @@ tear-down-kind2-env:
 test-e2e-local-fast: $(GINKGO)
 	./hack/test-e2e-local.sh --label-filter "Shoot && fast"
 
+test-e2e-local-cpm: $(GINKGO)
+	./hack/test-e2e-local.sh --label-filter "CPM"
+
 test-e2e-local: $(GINKGO)
 	@# run at maximum 5 tests in parallel for now until we have better experience of how much load a single prow pod can take
-	./hack/test-e2e-local.sh --procs=5
+	./hack/test-e2e-local.sh --procs=5 --label-filter="!CPM"
```

Comment:
What about adding a `default` label to all – well – default tests instead of filtering something out?